src/server/middleware/reactApplication/index.js

```diff
@@ -68,6 +72,8 @@ function reactApplicationMiddleware(request, response) {
     // html, and then the client bundle can use this data to know which chunks/
     // modules need to be rehydrated prior to the application being rendered.
     codeSplitState: codeSplitContext.getState(),
+    // Exposed styled-components string to the HTML.
+    styledComponents: styles,
```

Comment:
You add that string as part of the options argument, named `styledComponents`. Why the name change?